src/mscorlib/src/System/Reflection/Emit/ILGenerator.cs

```diff
@@ -1174,6 +1174,25 @@ public virtual void ThrowException(Type excType)
             this.Emit(OpCodes.Throw);
         }
 
+        private static Type GetConsoleType()
+        {
+#if PLATFORM_UNIX
+            Type t = null;
+            try
+            {
+                t = Type.GetType("System.Console, System.Console, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a", throwOnError: false);
+            }
+            catch { } // even with throwOnError:false, could still throw for a variety of reasons
```

Comment:
I would think that the "variety of reasons" kind of exceptions should be allowed to propagate. Or is there anything particular that we want to catch here?